config/standalone.example.toml

```diff
@@ -177,3 +177,15 @@ parallel_scan_channel_size = 32
 # otlp_endpoint = "localhost:4317"
 # The percentage of tracing will be sampled and exported. Valid range `[0, 1]`, 1 means all traces are sampled, 0 means all traces are not sampled, the default value is 1. ratio > 1 are treated as 1. Fractions < 0 are treated as 0
 # tracing_sample_ratio = 1.0
+
+# standalone/frontend/datanode/metasrv export the metrics generated by itself
+# send metrics to Prometheus remote-write compatible receiver (e.g. `greptimedb`)
+# [remote_write]
+# whether enable export remote_write, default is false
+# enable = false
+# The url of remote write endpoint.
+# Taking greptimedb as an example, for `standalone` deployed under the default configuration.
+# The user can create a database called `system` in the db and export the metric to `http://127.0.0.1:4000/v1/prometheus/write?db=system`
+# endpoint = "http://127.0.0.1:4000/v1/prometheus/write?db=system"
```

Comment:
Standalone and frontend should be able to avoid the network stack by invoking themselves.

And prefer to config endpoint, database, extra header etc separately.

Reply:
separate the config, another pr solve this
> standalone and frontend should be able to avoid the network stack